### PR TITLE
Added: Fluid content secondary field

### PR DESCRIPTION
**Pull request by TheBB**

### CODE REVIEW

**Comment by TheBB** [comment]
jenkins build this please

**Comment by TheBB** [comment]
Apparently origin was pointing here. Fixed that for next time. :-)

**Comment by TheBB** [comment]
jenkins build this please

File: PoroElastic/PoroElasticity.C [hunk 1]
```diff
@@ -534,17 +544,21 @@ bool PoroElasticity::evalSol (Vector& s, const FiniteElement& fe,
   Vector eV;
   utl::gather(MNPC,npv,primsol.front(),eV);
 
-  Vector disp(nsd * fe.N.size());
-  for (size_t i = 0; i < nsd; i++)
-    for (size_t a = 0; a < fe.N.size(); a++)
-      disp[nsd*a+i] = eV[npv*a+i];
+  Vector displacement(nsd * fe.N.size());
+  Vector pressure(fe.N.size());
```
Comment (akva2) [inline PoroElastic/PoroElasticity.C:548]: here you could do
```
double pressure = eV.dot(fe.N, nsd, nsd+1);
```

File: PoroElastic/PoroElasticity.C [hunk 2]
```diff
@@ -568,6 +582,16 @@ bool PoroElasticity::evalSol (Vector& s, const FiniteElement& fe,
   const RealArray& sig = sigma;
   s.insert(s.end(),sig.begin(),sig.end());
 
+  double pressure = press.dot(fe.N);
```
Comment (akva2) [inline PoroElastic/PoroElasticity.C:585]: isn't it simpler to just pass press here ?

Comment (TheBB) [reply PoroElastic/PoroElasticity.C]: Updated. Is this more like what you had in mind?

**Comment by TheBB** [comment]
jenkins build this please

File: PoroElastic/PoroElasticity.C (hunk 2 above)
Comment (akva2) [reply PoroElastic/PoroElasticity.C]: yup.

**Pull request merged**